webapp/components/abstract-test-file-results-table.js

```diff
@@ -142,6 +135,17 @@ class AbstractTestFileResultsTable extends WPTColors(TestRunsBase) {
     }
     return 'result';
   }
+
+  computeAnalyzerURL(screenshots) {
+    if (!screenshots) {
+      throw "empty screenshots";
+    }
+    const url = new URL('/analyzer', window.location);
+    for (const [_, sha] of screenshots) {
```

Comment:
```suggestion
    for (const sha of screenshots.values()) {
```

(`screenshots` is a Map)